index.md

```diff
@@ -18,10 +18,14 @@ The WSTG is a comprehensive guide to testing the security of web applications an
 
 ## Stable
 
-View the always-current stable version at [/stable](/stable).
+View the always-current stable version at [/stable](stable/).
```

Comment:
Should this not be the base url as well? Similar for the latest links

Reply:
It could use it but theyre static locations so using a relative URL is fine. Whereas with the banner it'll be displayed on all kinds of paths so we need it to link specifically not relatively.

Reply:
I was thinking more to it being related to local build issues

Reply:
Did you test it locally and have a problem? It should have been fine. I'll re-test.

Reply:
Works fine locally, logically should be fine deployed as well.